### PR TITLE
vet-clinic: Query and update

### PR DESCRIPTION
**Pull request by Ceemos96**
### In this milestone, I : 
- Added a species column to the table
- Carried out multiple transactions and queries to the database.
- Add species column and set as "unspecified", then rolled back the changes.
  - Screenshot (126)
- Set species for animals with names ending with 'mon' to be 'digimon' and others to be 'pokemon'.
  - Screenshot (127)
- Delete animals table and rollback changes
  - Screenshot (128)
- update animal weights and use savepoints in a transaction
  - Screenshot (129)
- Query database for requested data
  - Screenshot (130)

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -7,3 +7,42 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
...
+ROLLBACK;
+
+/* transaction*/
+BEGIN;
+UPDATE animals
+SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals
+SET species = 'pokemon' WHERE name NOT LIKE '%mon';
+COMMIT;
+
```
Comment (thecodechaser) [inline queries.sql:25]: - [ ] Please verify the change was made after the update and rollback as requested in the project requirements.
  - Hint: `SELECT species from animals;`

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Ceemos96👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Ceemos96) [reply queries.sql:25]: Hello,
Kindly view the screenshot attached to the PR. It shows the full table including the species column. This clearly exhibits the changes before and after they were made.

Comment (thecodechaser) [reply queries.sql:25]: @Ceemos96 Please also include it in the queries file.

Comment (Ceemos96) [reply queries.sql:25]: @thecodechaser It is all there. kindly view the documents thoroughly before make your reviews.

Comment (thecodechaser) [reply queries.sql:25]: @Ceemos96 No it's not there, Please review your file.
After the update, add commands to verify that the change was made. I already provided you with a hint

Comment (Ceemos96) [reply queries.sql:25]: Here is a copy of the attached screenshot in the PR description. 
It clearly shows the animals table including the species column,  it also shows that all rows have the 'unspecified' value. Futhermore, it shows that i have rolled back the changes as required by the project.

Screenshot (126)

**Review by KayLemba — APPROVED** [review]
[# STATUS: APPROVED :green_circle: :clap: 

Hello  @Ceemos96      , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- [x]  Your repo is professional 
- [x] Gitflow is followed
- [x] Descriptive P.R
- [x] Descriptive README file




 

- [x] No Linter Errors







Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]

**Pull request merged**